pkgs/by-name/le/lenmus/package.nix

```
@@ -23,23 +23,15 @@
 
 stdenv.mkDerivation (finalAttrs: {
   pname = "lenmus";
-  version = "6.0.1";
+  version = "6.0.1-unstable-2023-05-14";
```

This date is wrong should probably be

```suggestion
  version = "6.0.1-unstable-2025-09-15";
```

---

Indeed. Not sure what happened there. :(